### PR TITLE
Enable holmes as pyroscope client and reports pprof event to pyroscope server

### PR DESCRIPTION
**Pull request by Jun10ng**
#104

### CODE REVIEW

**Review by doujiang24 — COMMENTED** [review]
Need some doc for this new reporter, otherwise, LGTM.

**Comment by Jun10ng** [comment]
@doujiang24

File: readme.md [hunk 1]
```diff
@@ -185,6 +185,13 @@ You can use `Reporter` to implement the following features:
 )
   
 ```
+
+#### Enable holmes as pyroscope client
+
+Holmes supports to upload your profile to [pyroscope](https://github.com/pyroscope-io/pyroscope) server. More details
```
Comment (doujiang24) [inline readme.md:191]: we need some simple doc, for how to enable this feature.
Assuming that people already have a pyroscope service.

**Comment by cch123** [comment]
Holmes uses a different approach from continuous profiling, I don't know whether it's proper to integrate the event triggered profile to pyroscope, have you already used it on your production env?

**Review by cch123 — APPROVED** [review]

**Pull request merged**